ping/_compositions/rust.toml

```diff
@@ -1,7 +1,11 @@
 [master]
-BinaryName = 'testplan_0500'
+BinaryName = 'testplan_0510'
```

Comment:
Neat trick. Just need to remember that this is not actually `v0.51.0` (yet).

Reply:
We should probably rename it to `testplan_master`. Really the name doesn't matter :)